### PR TITLE
HOCS-3976: remove case viewed audits

### PR DESCRIPTION
**Pull request by nickmcmahon01**
Hocs-3976: remove call causing case viewed audits when not needed

### CODE REVIEW

File: src/main/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspect.java [hunk 1]
```diff
@@ -82,9 +84,9 @@ private void verifyAllocatedToTeam(UUID caseUUID, UUID stageUUID) {
     }
 
     private boolean proceedIfUserTeamIsAdminForCaseType(UUID caseUUID) {
-        String stageType = caseworkClient.getCase(caseUUID).getType();
+        String shortCode = caseUUID.toString().substring(34);
```
Comment (LiamMacP) [inline src/main/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspect.java:87]: This still feels like a hack, can we add something to review this...

File: src/test/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspectTest.java [hunk 2]
```diff
@@ -25,6 +26,9 @@ public class AllocatedAspectTest {
     @Mock(answer = Answers.RETURNS_DEEP_STUBS)
     private CaseworkClient caseworkClient;
 
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
```
Comment (LiamMacP) [inline src/test/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspectTest.java:29]: Does this need `(answer = Answers.RETURNS_DEEP_STUBS)`?

Comment (nickmcmahon01) [reply src/test/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspectTest.java:29]: 👻 whooo knowws, i'll take it out

File: src/main/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspect.java (hunk 1 above)
Comment (nickmcmahon01) [reply src/main/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspect.java:87]: there is an endpoint on casework we could add to workflow, but that endpoint calls the info service one. so we can unpick it as one task.

**Review by LiamMacP — APPROVED** [review]

**Pull request merged**